LayoutTests/http/tests/site-isolation/edge-sampling-commit-root-frame-load.html

```diff
@@ -0,0 +1,55 @@
+<!-- webkit-test-runner [ SiteIsolationEnabled=true pageTopColorSamplingEnabled=true ] -->
+<head>
+    <script src="/js-test-resources/ui-helper.js"></script>
+    <script>
+        if (window.testRunner) { testRunner.waitUntilDone(); testRunner.dumpAsText() }
+        async function runTest() {
+            if (window.testRunner && !testRunner.contentInsetBackgroundFillEnabled()) {
```

Comment:
Is there a reason why we're early returning here if the feature is off at the system level, rather than using the test option to enable it everywhere?

Reply:
That didn't work for an unknown reason on shipping OSes.

Reply:
Oh, I think I see it.